config/schedule.rb

```diff
@@ -1,3 +1,6 @@
+rails = File.join Dir.getwd, "config", "environment.rb"
+require rails
```

Comment:
If we hit `http://localhost/` we get an https redirect though `https://localhost/` will give us a certificate error (which we could ignore/override). And it would mean hardcoding the path in this file. Workable?

We could possibly hit the unicorn unix domain socket, though that would be fork-specific, and we would need to upgrade the version of `curl` we have on the production servers to get that feature (curl 7.40.0+).